### ISSUE TITLE
Module '"rn-id-blurview"' has no exported member 'BlurView'.

### ISSUE THREAD

**Issue by elgambet** [idrisssakhi/blurView#10]
I have problems with typescript:

```js
// Module '"rn-id-blurview"' has no exported member 'BlurView'.

import {BlurView} from 'rn-id-blurview';
```

if I add a @ts-ignore to the import it works fine:

```js
//@ts-ignore
import {BlurView} from 'rn-id-blurview';
```

Steps to reproduce:

```sh
# Create app from ts template
npx create-react-app --template typescript test
# Add dependency
cd test && yarn add rn-id-blurview
# Edit App.tsx with import {BlurView} from 'rn-id-blurview';
```

Example:

```sh
# Clone project
yarn clone git@github.com:elgambet/rn-id-blurview-ts-error.git
# Install dependencies
cd rn-id-blurview-ts-error && yarn install
# Open src/App.tsx file
```

**Comment by Orange9000** [comment]
Same on react-native 0.69.1

**Comment by idrisssakhi** [comment]
@elgambet @Orange9000 would you test version 1.2.2 and tell me if the issue is always present. Thank you in advance

**Comment by slauzinho** [comment]
I can confirm this appear to be fixed in the version `1.2.2`
